content/doc/developer/internationalization/_chapter.yml

```diff
@@ -4,6 +4,7 @@ sections:
 - i18n-jelly-views
 - i18n-groovy-views
 - advanced-localization
+- new-internationalization
```

Comment:
I am not sure about the "new" prefix. Why don't we deduplicate the content with existing sections here and introduce an "introduction" chapter in the beginning

Reply:
So according to you, we should add an introduction section about internationalization in the existing content.

Reply:
@oleg-nenashev Is the above response to you comment is correct?

Reply:
Yes, thanks!